MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/offline/OfflineManager.java

```diff
@@ -629,14 +628,6 @@ public void run() {
    */
   public void createOfflineRegion(@NonNull OfflineRegionDefinition definition, @NonNull byte[] metadata,
                                   @NonNull final CreateOfflineRegionCallback callback) {
-    if (!isValidOfflineRegionDefinition(definition)) {
```

Comment:
shall we add an instrumentation test for a region across the 180th meridian?

Reply:
sure but that will only work once we upgrade underlying core release